packages/lab/src/form-field/FormHelperText.css

```diff
@@ -33,7 +33,7 @@
 }
 
 /* Class applied to the helper text container */
-.saltFormFieldHelperText {
```

Comment:
i think we shouldn't touch this component at all

Reply:
I've kept this because the name of the file is FormHelperText and so keeping patterns the class name should match the file name
It is also making storybook confused as these styles are overriding the form field next helper text padding due to having the same class name (idk why that's happening as that css file is not imported, i guess it's all bundled together but ideally that shouldnt happen?)